src/app/routes/utils/sharedDataTransformers/augmentWithTimestamp/index.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-use-before-define */
```

Comment:
Can this now be removed?

Reply:
No, I just tried to remove it and still got errors